dashboard/test/helpers/pd/workshop_survey_results_helper_test.rb

```diff
@@ -132,15 +62,4 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
     result_hash = summarize_workshop_surveys(workshops: [workshop_1, workshop_2], facilitator_breakdown: false)
     assert_equal 2, result_hash[:how_clearly_presented]
   end
-
-  test 'get an error if summarizing a mix of workshop surveys' do
-    local_workshop = create(:pd_local_summer_workshop_survey).pd_enrollment.workshop
-    local_workshop.update(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP)
-
-    teachercon = create(:pd_teachercon_survey).pd_enrollment.workshop
-    teachercon.update(course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_TEACHER_CON)
-    assert_raise RuntimeError do
-      summarize_workshop_surveys(workshops: [local_workshop, teachercon])
-    end
-  end
```

Comment:
Based on [the comment](AAAA) in the original method, it says the method works on arrays where everything is either a teachercon survey or workshop survey. Since this PR removes the workshop survey option, there's no need to have this test to check if there's a mix of them.